app/models/legal_aid_application.rb

```diff
@@ -176,6 +176,10 @@ def calculation_date
     LegalAidApplications::CalculationDateService.call(self)
   end
 
+  def year_to_calculation_date
+    [calculation_date - 1.year + 1.day, calculation_date]
```

Comment:
Hi, just wondering why `1.year`, i thought it'd be 3 months of transactions? Also how comes the `1.day` addition?

Reply:
ohh, is this related to "student loan" being annual? If so will that be the only type of irregular income (annual ones)?

Reply:
yes - they declare the total amount they have received in the past year.  The means calculation works out what the monthly figure is, but on the certificate, we're just playing back to them what they have declared.  So the period is different (1 year as opposed to three months) .